content/02.main-text.md

```diff
@@ -220,6 +220,12 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+Pandoc brands itself as a "universal document converter", and can, as of version 2.7, convert from any of 32 input formats to any of 51 output formats.
+Thanks to its versatility and active development since 2006, Pandoc enjoys a large userbase across many disciplines and applications.
+Its filter interface enables adding custom functionality with community-developed programs.
+We are [prototyping](https://github.com/manubot/manubot/pull/99 "Create a pandoc-manubot-cite filter for pandoc") a citation-by-identifier filter.
```

Comment:
We could call this a `a Manubot-based citation-by-identifier filter` to add the Manubot branding and make it obvious how this paragraph about Pandoc connects to the prior paragraph.